examples/route_guide/server/server.go

```diff
@@ -45,13 +45,14 @@ import (
 	"github.com/golang/protobuf/proto"
 
 	pb "google.golang.org/grpc/examples/route_guide/routeguide"
+	"google.golang.org/grpc/examples/route_guide/testdata"
```

Comment:
why not just use the grpc/testdata package?

Reply:
I wanted to keep example data separate from test data.

Also I renamed the package to `exampledata`.